apps/sample-app/src/components/utils.tsx

```diff
@@ -0,0 +1,28 @@
+import { teamsDarkTheme, teamsHighContrastTheme, teamsLightTheme, Theme } from '@fluentui/react-components';
+import { app } from '@microsoft/teams-js';
+
+export const getTheme = (themeNow: string): Theme => {
+  switch (themeNow) {
+    case 'dark':
+      return teamsDarkTheme;
+      break;
+    case 'contrast':
+      return teamsHighContrastTheme;
+      break;
+    default:
+      return teamsLightTheme;
+  }
+};
+export function appInitializationFailed(): void {
+  app.notifyFailure({
+    reason: app.FailedReason.Other,
+    message: 'App initialization failed',
+  });
+}
+
+export function getDates(): [string, string] {
+  const current = new Date();
+  const currDate = current.getFullYear() + '-' + (current.getMonth() + 1) + '-' + current.getDate();
+  const tomorrowDate = current.getFullYear() + '-' + (current.getMonth() + 1) + '-' + (current.getDate() + 1);
```

Comment:
What happens if currDate is the last day of the month? e.g. January 31st?

Reply:
Oh right! Thanks for pointing this out to me. I have changed the code to use tomorrow.setDate(current.getDate() + 1); (getDate will get the right 'day number', and setDate will get the month and year accordingly)